source/dubregistry/web.d

```diff
@@ -100,13 +114,22 @@ class DubRegistryWebFrontend {
 			default: std.algorithm.sorting.sort!compare(packages); break;
 			case "name": std.algorithm.sorting.sort!((a, b) => a.name < b.name)(packages); break;
 			case "score": std.algorithm.sorting.sort!((a, b) => a.stats.score > b.stats.score)(packages); break;
-			case "added": std.algorithm.sorting.sort!((a, b) => getDateAdded(a) > getDateAdded(b))(packages); break;
+			case "added": std.algorithm.sorting.sort!((a, b) => a.dateAdded > b.dateAdded)(packages); break;
 		}
 
+		// limit package list to current page
+		size_t package_count = packages.length;
+		packages = packages[min(skip, $) .. min(skip + limit, $)];
+
+		// collect package infos
+		Json[string] infos;
+		foreach (p; m_registry.getPackageInfos(packages.map!(p => p.name).array))
+			infos[p.info["name"].opt!string] = p.info;
+
 		Info info;
-		info.packageCount = packages.length;
-		info.packages = packages[min(skip, $) .. min(skip + limit, $)]
-			.map!(p => Info.Package(p.stats, m_registry.getPackageInfo(p, false).info))
+		info.packageCount = package_count;
```

Comment:
Ehm mixing styles couldn't be more obvious ;-)

Reply:
You are right in so far that this should be `pcnt` or similar according to the vibe.d style guide. `package_count` would be the correct style for parameters, though.